src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBarracks.java

```diff
@@ -111,21 +113,24 @@ public void onUpgradeComplete(final int newLevel)
         }
 
         EnumFacing facing = EnumFacing.NORTH;
+
+        final int offset = getStyle().toLowerCase(Locale.ENGLISH).contains("birch") ? TOWER_OFFSET + 1: TOWER_OFFSET;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS864)